### PR TITLE
Feature/new classifiers

### PR DESCRIPTION
**Pull request by rcehemann**
image

Added three new classifiers and changed predict/train scripts accordingly. Also added a reclassify script in py_feels/binary to update the database when changes are made to classifiers. Needs to be run manually. I've also included a pickled dictionary with paths to saved classifiers, which requires loading the 'pickle' module. The scikit 'load' function uses pickle so this shouldn't cause any problems but if it does 'conda install pickle' should fix it.

I changed ex_feels/lib/ex_feels_web/templates/home/ so that Tweets now display with a twitter-like blue background and the feel section gets color-coded by sentiment. I wasn't able to merge the tweet sentiments into a table below tweet text, but when we get that done the blue background will distinguish tweet from sentiments.

I bravely made a change to feel.ex, commenting out the 'unique' filter for tweet_id, in order to see the sentiments from all four classifiers.

Removed binary_classifier.pckl; models are now stored as  __ class __ . __ name __ . pckl. This might be an issue (along with the classifier label) if we start trying to compare different parameter sets for the same class of classifier. We can cross this bridge if and when we get to it.

Added some more documentation to the training script in hopes that you or Jason have an easier time understanding the workflow and adding your own classifiers if you want. Warning: neural networks take FOREVER to train with scikit, so don't go adding a twelve-layer deep NN unless you plan on running the training script while you're at work all week. If we want to get fancy, I can train various types of neural networks on my GPU with the TensorFlow package much quicker. However, this will require downgrading to python 3.5 on the server.

If you want to set up the html table of feels, you can merge this in and submit your own PR. If you'd rather point out what I should do (hence the "help wanted" label), I can update the PR tonight or this week. I'll be gone for most of today 😭 

You'll notice a change to package.json involving node-sass. I had to manually install this because "npm install" was failing; not sure what was going on. Hopefully I didn't break anything.

### CODE REVIEW

File: ex_feels/assets/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "bulma": "^0.6.1",
+    "node-sass": "^4.7.2",
```
Comment (thekeele) [inline ex_feels/assets/package.json]: Remove this. Use `lts boron (6.12.3)`, install via `nvm`

**Comment by rcehemann** [comment]
all changes we discussed have now been made. next up: automation followed by some time-series visuals? 🙏
